src/app.tsx

```diff
@@ -19,9 +25,36 @@ export const App = () => {
     getPokemon().then(setItems);
   }, []);
 
+  const searchValueChangeHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchValue(e.target.value);
+  };
+
+  const singlePokemonClickHandler = (e: Event) => {
+    e.preventDefault();
+    setSinglePokemon([{}]);
+    setError(undefined);
+    getSinglePokemon(searchValue)
+      .then(setSinglePokemon)
+      .catch((error) => {
+        setError(error.message);
+      });
+  };
+
   return (
     <div>
       <div className="App">
+        <SinglePokemonForm
+          value={searchValue}
+          change={searchValueChangeHandler}
+          click={singlePokemonClickHandler}
+        />
+
+        {singlePokemon ? (
+          <SinglePokemonComponent singlePokemonData={singlePokemon} />
```

Comment:
re: https://github.com/Parkcrest/Pokedex/pull/8/files#r1068293096

```suggestion
          <SinglePokemonComponent {...singlePokemon} />
```